bc-express/migrations/20170602214520-create-users.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      firstName: {
+        type: Sequelize.STRING
+      },
+      lastName: {
+        type: Sequelize.STRING
+      },
+      email: {
+        type: Sequelize.STRING
+      },
+      neighborhood: {
+        type: Sequelize.STRING
+      },
+      password: {
```

Comment:
looks like plain passwords are being stored, are you doing encryption of password as a separate story? (I think that's a good idea, just checking)